DOCKER.md

```diff
@@ -31,3 +31,12 @@ $ docker run --rm -it -v $PWD/test:/workspace --entrypoint bash jenkinsfile-runn
 root@dec4c0f12478:/src# cp -r /app/jenkins /tmp/jenkins
 root@dec4c0f12478:/src# /app/bin/jenkinsfile-runner -w /tmp/jenkins -p /usr/share/jenkins/ref/plugins -f /workspace
 ```
+
+## Debug
+In case you want to debug Jenkinsfile Runner the recommendation is to use the image built following the steps mentioned in the section above.
+
+Then, set the `DEBUG` environment variable and expose the port where to connect the remote debug
```

Comment:
I think it's worth specifying that the port to be exposed is 5005, given it cannot be changed.

Reply:
It can be changed on the user side. `docker run -p 1234:5005 -e DEBUG=1`

Reply:
Oleg's right. The internal port is always 5005, but users can specify any other port for their remote debug.

Reply:
Yes, I know, but I think it's worth mentioning the one that can't be changed.

Reply:
Would it work for you? 
```suggestion
Then, set the `DEBUG` environment variable and expose the port 5005 where to connect the remote debug
```
In my opinion this might be ambiguous about the ports to expose/configure.

Reply:
I prefer it but if you don't feel sure enough or think it's confusing, I'm fine with it too.